### DISCUSSION TITLE
What is that picture?

### DISCUSSION THREAD

**Discussion by r0hitm** [Yisus7u7/termux-desktop-xfce#3] [General]
Hi @Yisus7u7 I was going through the README, and I found [this](https://github.com/Yisus7u7/termux-desktop-xfce/blob/main/fotos/play_music.png) amazing wallpaper. I really tried to find it but could not. Do you have this? Would you mind sharing it? It is so good!
☮️

**Comment by JesusChapman** [comment]
If this, right clik on the desktop and then click on desktop settings, there will come a lot of wallpapers, among those is that background
